install/0000_90_cluster-version-operator_00_servicemonitor.yaml

```diff
@@ -16,3 +16,42 @@ spec:
   selector:
     matchLabels:
       k8s-app: cluster-version-operator
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    k8s-app: cluster-version-operator
+  name: cluster-version-operator
+  namespace: openshift-cluster-version
+spec:
+  groups:
+  - name: cluster-version
+    rules:
+    - alert: ClusterVersionOperatorDown
+      annotations:
+        message: ClusterVersionOperator has disappeared from Prometheus target discovery.
```

Comment:
Probably better to indicate why someone might care as well as possible causes since this critical.  “Operator may be down or disabled, cluster will not be kept up to date and upgrades will not be possible”

Reply:
@smarterclayton done with https://github.com/openshift/cluster-version-operator/pull/232/commits/4a9719f52fb3a4b5fe44eed7bf8f68adb58ec276